Updated data vis slides

mfiorina opened the pull request:
Changes made:
- Added suggested `use_course()` slide structure. Attempt to address #120.
- Changed language for exercise 1. Closes #117.
- Changed inclusion of the `latam` variable to occur prior to plot creation. Closes #96.
- `flair` package from CRAN doesn't work with r 4.3.2. Updated .Rmd to pull from GitHub. Hopefully this doesn't cause issues, although if it does, the package use isn't crucial, just nice.

ramarty commented:
1. On slide 24, maybe add `+geom_point()`, just so its similar structure to next one and previous code.
2. On slide 26, maybe simplify this: `mutate(latam = ifelse(region == "Latin America and Caribbean", TRUE, FALSE))` to just `mutate(region == "Latin America and Caribbean")`. Although maybe a reason not to simplify?
3. Not sure if there's a good way to fit this, but for changing scales, one thing that may be kinda common is to have a variable that is a proportion that you want to represent as a percent. Not sure if we can fit using this into an example: `scale_y_continuous(labels = scales::percent)` (or more simply could link to this as a resource maybe? https://thomasadventure.blog/posts/ggplot2-percentage-scale/)
4. Maybe on additional thing that could be helpful to include is using scale_[x/y]_continuous to change breaks, labels, and range? For example, if data shows something from 0 to 70%, may want to change plot to show up to 100%. Also may want to reduce/increase number of labels. Although realize there's about 453,273 different things we can teach about ggplot so we need to limit -- but this potentially may be common-ish enough to reference?

mfiorina commented:
Thank you @ramarty! I've implemented the following changes:
- Corrected lack of `geom_point()` in slide 26
- Implemented suggested change in slide 36
- Could find a good use case with the current dataset for `scales::percent`, but something I'll keep in mind. Changed use of `scale_log_10()` to `scale_x_continuous()` which I agree is more commonly used and useful
